File: templates/python/es/text/code/cornerBeepers10.py
# Archivo: CornerFive Conos .py
# -----------------------------
# conos cinco conos en cada esquina
from karel.stanfordkarel import *
def main():
   # Repita una vez para cada esquina
   for i in range(4):
      poner_cinco_conos()
      moverse_a_la_siguiente_esquina()

# reposicionar karel a la siguiente esquina
def moverse_a_la_siguiente_esquina() :
   moverse()
   moverse()
   moverse()
   girar_izquierda()

# lugares 5 conos usando un ciclo for
def poner_cinco_conos() :
   for i in range(5):
   	poner_cono()

File: templates/java/es/text/code/cornerBeepers10.java
/*
 * Archivo: EsquinaCincoConos.java
 * ------------------------------
 * conos cinco conos en cada esquina.
 */
import stanford.karel.*;
public class EsquinaCincoConos extends Karel {

   public void run() {
      /* Repetir una vez por cada esquina. */
      for(int i = 0; i < 4; i++) {
         ponerCincoConos();
         moverseALaEsquinaSiguiente();
      }
   } 

   /* reposicionar karel a la siguiente esquina */
   public void moverseALaEsquinaSiguiente() {
      moverse();
      moverse();
      moverse();
      girarIzquierda();
   }

   /* coloca 5 conos usando un ciclo for */
   public void ponerCincoConos() {
      for(int i = 0; i < 5; i++) {
      	ponerCono();
      }
   }
}